packages/flutter_tools/test/host_cross_arch.shard/ios_content_validation_test.dart

```diff
@@ -358,3 +357,46 @@ void main() {
      timeout: const Timeout(Duration(minutes: 7))
   );
 }
+
+bool _containsBitcode(String pathToBinary, ProcessManager processManager) {
```

Comment:
This was moved from the now-deleted `darwin_common.dart`, bitcode isn't a thing on macOS, so it's not shared. I don't know why I ever added the check to the macOS validation test.